py/examples/device_tapdetector_example/README.md

```diff
@@ -0,0 +1,31 @@
+
+### Completing the Tapping Procedure:
+A prerequisite to using the tap detector is the proper collection of data. This section outlines the methods that should be use to collect the data.
+
+Data should be collected from two sources simultaneously. The following is a set of generic steps to take for data collection of any two devices to use in the tap detector, you may need to take additional steps depending on the devices you are using and the best practices for using those devices.
+
+1. **Setup your Environment(s)** - Ensure that you have a working environment for both devices you plan to use that allows you to record data.
+1. **Open recording software(s)** - Start the data collection software for each device. Ensure that both devices have properly connected to their respective data collection software and are properly configured.
+1. **Begin data recording** - Begin recording data on both devices. It is recommened to try and begin recording on both devices simultaneously, as this will make validation that the files have lined up properly easier, but it is not strictly required.
+1. **Get devices into position** - Pick up both devices and hold them still, allowing the data settle to a baseline for at least 30 seconds. Do not make any sudden movements during this time.
+1. **Perform the taps** - Quickly and firmly hit the devices against each other 3 times, making contact approximately once per second. The taps need to be strong enough to make a noticeable "spike" in the accelerometer data. The more pronounced and clear the taps are as compared to the baseline data and other noise that may be surrounding the taps, the better the tap detector will be able to identify the taps. **WARNING: While hitting the devices together strongly is important, take care to not damage the devices. For example, tap a connector on one board against the side of another board, avoid hitting chips or other electronics on either board to prevent damage.**
+1. **Return to Baseline** - Allow the data to return to baseline again by holding the devices still for another 30 seconds. Steps 4 and 5 may be repeated as necessary to ensure sufficient data collection. It is recommened to collect at least 2 sets of taps.
+1. **End data recording** - End the recording on both devices. Perform any processing necessary on the data. Some devices may record data in a format the can be immediately used, but some devices (such as EmotiBit) require a parsing/processing step. The tap detector requires that the data being input has the Acceleration Data and Timestamps available. 
+
+### All Arguments:
```

Comment:
I think this is very useful information when dealing with CLI. maybe move this up around where you introduce the sample example?

Reply:
Have now also included this further up.